GrandCentralBoard/Widgets/Harvest/OAuthCredentials.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  OAuthCredentials.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+
+
+struct OAuthCredentials {
+    let accessToken: String
+    let refreshToken: String
+    let expirationTime: NSTimeInterval
+
+    var isTokenExpired: Bool {
+        return NSDate().timeIntervalSince1970 > expirationTime
```

Comment:
I used the AccessToken class to store the access token and expiration date, so that code no longer exists. However, it was easier to store the time interval since NSDate doesn't seem to have any good comparison methods, so I'd have to compare the intervals anyway.